candidate_models/base_models/__init__.py

```diff
@@ -268,8 +267,8 @@ def vonecornet(model_name='cornets'):
     from model_tools.activations.pytorch import load_preprocess_images
     preprocessing = functools.partial(load_preprocess_images, image_size=224,
                                       normalize_mean=(0.5, 0.5, 0.5), normalize_std=(0.5, 0.5, 0.5))
-    from candidate_models.base_models.stochastic import StochasticTemporalPytorchWrapper
-    wrapper = StochasticTemporalPytorchWrapper(identifier='vone'+model_name, model=model, preprocessing=preprocessing)
+    from candidate_models.base_models.cornet import TemporalPytorchWrapper
+    wrapper = TemporalPytorchWrapper(identifier='vone'+model_name, model=model, preprocessing=preprocessing)
     wrapper.image_size = 224
     return wrapper
```

Comment:
if I understand right, VOne{CORnet,Resnet} are now exclusively defined in the brain model pool no?

Reply:
there are 3 new models on the brain model pool: 'VOneCORnet-S', 'voneresnet-50' and 'voneresnet-50-robust. I don't understand what you mean by exclusively defined. The base model versions of the 'voneresnet-50' and 'voneresnet-50-robust' can also be accessed.

Reply:
I removed them from the base model pool in my commitment and added them to a StochasticModelPool. They should not be in the base model bool.

Reply:
I believe you removed them from the list (https://github.com/brain-score/candidate_models/pull/70/files#diff-30b56a69b0b5a99bcaf36381fc10683e55aff0b7a42516cd0ee8f6d1ff9c12a0L391-L393) but the function definitions are still there (https://github.com/brain-score/candidate_models/pull/70/files#diff-30b56a69b0b5a99bcaf36381fc10683e55aff0b7a42516cd0ee8f6d1ff9c12a0R251-R273, lines 251 to 274 in `base_models/__init__.py`)

Reply:
bump @tiagogmarques